README.md

```diff
@@ -28,38 +28,15 @@ Clone the three required repositories by
     git clone https://github.com/MetabolicAtlas/data-files && pushd data-files; git lfs pull; popd
 
 
-Go to the repository `data-generation` and follow the
-[instructions](https://github.com/MetabolicAtlas/data-generation#readme) on how to generate the data files required by MetabolicAtlas.
+Go to the repository `data-generation` and follow the [instructions](https://github.com/MetabolicAtlas/data-generation#readme) on how to generate the data files required by Metabolic Atlas.
 
-In the folder `MetabolicAtlas` that has been cloned, add a `.env` file based on the `.env.sample` file:
+In the folder `MetabolicAtlas` that has been cloned, add a `env-local.env` file based on the `env-local.env.sample` file:
```

Comment:
This question applies for the rest of the PR too. Could we do without the `env-` prefix? So we would have `local.env` and `prod.env` for example.

Reply:
Ah ok, hadn't thought about that at all. In my file browser I usually display things grouped by "Kind"anyways. Personally I would value less redundant file name parts more than this detail but it's not a big deal. I think it's overkill to have a folder for this now too.

<img width="385" alt="image" src="https://user-images.githubusercontent.com/423498/146040351-3e36a552-28d8-4a13-90b4-18b0d494eead.png">

Reply:
I didn't even think of the option of grouping files like that with Finder, I was just thinking of my own use-case in VS Code and imagined it would be quite common for all alphabetically-sorted terminal-based scenarios as well
<img width="214" alt="image" src="https://user-images.githubusercontent.com/23480589/146067025-f21c1968-3110-457e-ac87-d84fb2df6462.png">
I feel we should go with the popular vote for this one instead of basing this on my own use-case.

Reply:
Feels like this is not too big of a deal. Let's abandon the change suggestion for now.